x/tss/types/params.go

```diff
@@ -24,6 +24,7 @@ var (
 	KeyExternalMultisigThreshold        = []byte("externalMultisigThreshold")
 	KeyMaxSignQueueSize                 = []byte("MaxSignQueueSize")
 	MaxSimultaneousSignShares           = []byte("MaxSimultaneousSignShares")
+	KeySignedBlocksWindow               = []byte("SignedBlocksWindow")
```

Comment:
Can we rename this to something else? Since there is already a param from cosmos sdk with this name.
```suggestion
	KeySignedBlocksWindow               = []byte("HeartBeatSignedBlocksWindow")
```

Reply:
That is a good point, but I'd rather not use "heartbeat" because this is meant to calculate the `missed-too-many-blocks` illegibility, which is not necessarily used exclusively in the context of heartbeats.

Reply:
Yeah, I'm open to better names. This is just the quick and dirty one that came to my mind :p

Reply:
renamed it to `TssSignedBlocksWindow`